tests/_files/tree_network_sim_expected_results/calc_temp_drop.py

```diff
@@ -0,0 +1,190 @@
+"""
+This script calculates the heat transfer at the consumer
+for a simple tree network.
+"""
+import os
+import pandas as pd
+import math
+import numpy as np
+
+path_file = os.path.dirname(__file__)
+path = os.path.abspath(os.path.join(path_file, os.pardir, os.pardir, os.pardir))
+input_data = os.path.join(path, 'examples', 'simulation', 'tree')
+result_path = os.path.join(path_file, 'sequences')
+
+if not os.path.exists(result_path):
+    os.mkdir(result_path)
+
+
+def read_data(input_value):
+    """
+    This function is reading the data of a csv with a name given as input value
+    """
+    return pd.read_csv(os.path.join(input_data, input_value + '.csv'), index_col=0)
+
+
+# Read input data for every csv component
+pipes = read_data('pipes')
+temp_drop = pd.read_csv(input_data + '/sequences/consumers-temperature_drop.csv')
+mass_flow = pd.read_csv(input_data + '/sequences/consumers-mass_flow.csv')
+
+# Constants for calculation
+t_env = 10                                                                 # [°C]
+t_prod_i = pd.DataFrame(data={'t_prod_i': 130 * np.ones(len(mass_flow))})  # [°C]
+c = 4190                                                                   # [J/kg*K]
+pi = math.pi
+
+# Initialize variables of type dataframe (needed for later calculations)
+U_spec, t_cons_i, t_cons_r, t_fork_r, Q_loss_i, Q_loss_r, Q_cons, Q_loss_glob = \
+    [pd.DataFrame() for variable in range(8)]
+
+# Adjust mass flows and temp drop to a dataframe containing all data in correct order
+# Get mass flows of all consumers
+mass_flow_total = mass_flow.iloc[:, 1:]
+# Rename the columns to pipes naming convention
+mass_flow_total.columns = ['1', '2']
+# Calculate producer mass flow as sum of consumer mass flows
+mass_flow_total['0'] = mass_flow_total['1'] + mass_flow_total['2']
+# Change order of columns for later calculation
+mass_flow_total = mass_flow_total[['0', '1', '2']]
+
+# Get temperature drop of all consumers
+temp_drop = temp_drop.iloc[:, 1:]
+# Rename the columns to pipes naming convention
+temp_drop = temp_drop.rename(columns={'0': '1', '1': '2'})
+
+
+def calc_temp_heat_loss(t_in, index):
+    """
+    This function calculates the pipe's outlet temperature
+    out of the inlet temperature due to heat losses
+    :param t_in:
```

Comment:
Please use numpy-style docstrings

Reply:
Done in https://github.com/oemof/DHNx/pull/39/commits/4cca64d587b6b43e4a97f5113a45a5967e442937 and https://github.com/oemof/DHNx/pull/39/commits/e560016e6905e6911c6816555fc640fa2bc57b39